lib/oli/rendering/content/selection.ex

```diff
@@ -94,7 +94,15 @@ defmodule Oli.Rendering.Content.Selection do
         %{}
 
       ids ->
-        Oli.Publishing.DeliveryResolver.from_resource_id(section_slug, ids)
+        revisions =
+          if Enum.any?(
```

Comment:
This selection renderer is unique - as it only ever gets called during page preview.  In normal delivery, by the time we get to rendering the activity bank selections have been "fulfilled" and have been replaced by actual activities.  For page preview, we do not fulfill bank selections, we instead leave them in the page model and this rendering code provides a summary of them to the author.  So that being said, we never want to use the `DeliveryResolver` in this code, we only want to resolve to the Authoring revisions. 

I suspect the use of DeliveryResolver was the problem and what was yielding the `nil` ids.  I think we should simplify this code to:

```
 Oli.Publishing.AuthoringResolver.from_resource_id(section_slug, ids). # This section_slug really is the project slug
|> Enum.filter(fn r -> !is_nil(r) end)
|> Enum.reduce(%{}, fn rev, m -> Map.put(m, rev.resource_id, rev.title) end)
```